core/sql/common/ExprNode.cpp

```diff
@@ -737,6 +737,16 @@ NABoolean OperatorType::match(OperatorTypeEnum wildcard) const
 	      return FALSE;
 	    }
 
+        case REL_ANY_EXTRACT:
+	  switch (op_)
+	    {
+	    case REL_FAST_EXTRACT:
```

Comment:
It's counterintuitive to me that a Hive Insert (or any Insert) is considered an Extract. "Extract" to me has the connotation of pulling something out, while "Insert" to me has the connotation of putting something in. Perhaps "Extract" in this context is in reference to a Trafodion object? But would a Hive Insert always be taking data from a Trafodion object? Or might we support Insert/Select from one Hive object to another?

Reply:
Thank you for the explanation. It's fine as it is.